### PR TITLE
oooof

### PR DESCRIPTION
**Pull request by Legoburner73**
hoster there so i can actually start it up LOL

### CODE REVIEW

**Comment by haykam821** [comment]
The corresponding server pull request: https://github.com/CommunityDiep/server/pull/8

File: client.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-"use strict";
+//"use strict";
```
Comment (haykam821) [inline client.js:1]: Why?

File: client.js [hunk 2]
```diff
@@ -238,7 +238,7 @@ class Player {
 	}
 }
 
-let angle = 0;
+var angle = 0;
```
Comment (haykam821) [inline client.js:241]: Why are we downgrading our beautiful `let`?

File: hoster.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+app.get("/functions.js", function (request, response) {
+    response.sendFile(__dirname + '/functions.js');
+});
+app.get("/client.css", function (request, response) {
+    response.sendFile(__dirname + '/client.css');
+});
+
+let listener = app.listen(process.env.PORT, function () {
+  console.log('Your app is listening on port ' + listener.address().port);
+});
```
Comment (haykam821) [inline hoster.js:21]: Nothing in this file is needed.

File: client.js (hunk 1 above)
Comment (Legoburner73) [reply client.js:1]: Thought it was causing issues

File: client.js (hunk 2 above)
Comment (Legoburner73) [reply client.js:241]: Also thought it was causing issues.

File: hoster.js (hunk 3 above)
Comment (Legoburner73) [reply hoster.js:21]: I needed a way to boot up client.

**Pull request merged**